build-support/bin/mypy.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import subprocess
+
+
+def main() -> None:
+  subprocess.run([
+    "./pants",
+    "--tag=+type_checked",
+    "mypy",
+    "src/python/pants::",
+    "tests/python/pants_test::",
+    "contrib::",
+    "build-support/bin::",
```

Comment:
Would there happen to be anything that we'd want to target for this that fall outside of these paths (ie. would `./pants --tag=+type_checked mypy ::` be preferable here since we're already being explicit on the tag)?

Reply:
Great suggestion! There are places we could currently be missing, like `build-support/travis` and `testprojects`.